tests/reftests/dune.inc

```diff
@@ -692,24 +728,6 @@
    %{targets}
    (run ./run.exe %{exe:../../src/client/opamMain.exe.exe} %{dep:list.unix.test} %{read-lines:testing-env}))))
 
-(rule
- (alias reftest-local-cache)
```

Comment:
why removing/integrate this perfectly good test?

Reply:
If this is just for sharing, could you split the commit into two:
* one that adds content to `local-cache.test`
* the next that moves `local-cache.test` to `url-section-checks.test`

That would make it easier to review and know what has changed.

I'm also unsure about the name `url-section-checks` doesn't tell me much personally. Especially if it still deals with checking the cache coherency I feel like the previous name was just fine for that

Reply:
Indeed, it is for sharing the setup. Commits are already splitted, see https://github.com/ocaml/opam/pull/5560/commits/a7517fd364583e61240ec8ce938d734b167551f8.
On the name, i'm not happy with the three new test names, but i don't have better names.

Reply:
updated